src/libraries/shims/ApiCompatBaseline.PreviousNetCoreApp.txt

```diff
@@ -1,4 +1,11 @@
+Compat issues with assembly mscorlib:
+CannotRemoveAttribute : Attribute 'System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute' exists on 'System.Delegate.CreateDelegate(System.Type, System.Type, System.String)' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute' exists on 'System.Delegate.CreateDelegate(System.Type, System.Type, System.String, System.Boolean)' in the contract but not the implementation.
+CannotRemoveAttribute : Attribute 'System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute' exists on 'System.Delegate.CreateDelegate(System.Type, System.Type, System.String, System.Boolean, System.Boolean)' in the contract but not the implementation.
 Compat issues with assembly netstandard:
+CannotRemoveAttribute : Attribute 'System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute' exists on 'System.Delegate.CreateDelegate(System.Type, System.Type, System.String)' in the contract but not the implementation.
```

Comment:
@Anipik @safern @ericstj - the `RequiresUnreferencedCodeAttribute` was introduced in `net5.0`. I'm not sure how it got into mscorlib and netstandard. Did we update those in the 5.0 timeframe?

Also, can you take a look at this change and tell me if it looks correct?